Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -1501,11 +1508,11 @@
                     "kind": {
                         "type": "string",
                         "enum": [
-                            "article", 
-                            "project", 
-                            "section", 
-                            "overview", 
-                            "symbol", 
+                            "article",
+                            "project",
+                            "section",
+                            "overview",
+                            "symbol",
```

Comment:
I think it would be better to leave this as-is just to not confuse the commit history.